apps/modernization-ui/src/routes/AppRoutes.tsx

```diff
@@ -29,7 +29,7 @@ const routing = [
         ]
     },
     { path: '*', element: <RedirectHome /> },
-    { path: 'expired', element: <Expired /> }
+    { path: '/logout', element: <Expired /> }
```

Comment:
issue:  This path can remain as `expired` the time out page is meant to be displayed when the NBS6 session is timed out.  Currently, when a session is timed out the user is navigated to the `/nbs/timeout` page.  There is another story to handle the stangling of the `/nbs/timeout` page that we replace it with the new time out page.

Reply:
fixed